hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/dispatch/ChildDispatchFactory.java

```diff
@@ -160,33 +158,31 @@ public Dispatch createChildDispatch(
             @NonNull final HandleContext.TransactionCategory category,
             @NonNull final ExternalizedRecordCustomizer customizer,
             @NonNull final SingleTransactionRecordBuilderImpl.ReversingBehavior reversingBehavior,
-            @NonNull final RecordListBuilder recordListBuilder,
             @NonNull final Configuration config,
             @NonNull final SavepointStackImpl stack,
             @NonNull final ReadableStoreFactory readableStoreFactory,
             @NonNull final NodeInfo creatorInfo,
             @NonNull final PlatformState platformState,
             @NonNull final HederaFunctionality topLevelFunction,
-            @NonNull final ThrottleAdviser throttleAdviser) {
-        final var preHandleResult =
-                dispatchPreHandleForChildTxn(txBody, syntheticPayerId, config, readableStoreFactory);
+            @NonNull final ThrottleAdviser throttleAdviser,
+            final Instant consensusNow) {
+        final var preHandleResult = preHandleChild(txBody, syntheticPayerId, config, readableStoreFactory);
```

Comment:
Since Dispatch has a `PreHandleResult` [here](https://github.com/hashgraph/hedera-services/blob/7de1e47e2bfa1eafe0ffbc4d9e676100c6a5e02f/hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/Dispatch.java#L207), it is computed inside `createChildDispatch`. Changed the javadoc.